e2e/fixtures/wizard.template.yaml

```diff
@@ -0,0 +1,29 @@
+HeadNode:
+  InstanceType: t2.micro
+  Networking:
+    SubnetId: subnet-006e97a9837b1710d
+  LocalStorage:
+    RootVolume:
+      VolumeType: gp3
+Scheduling:
+  Scheduler: slurm
+  SlurmQueues:
+    - Name: queue0
+      AllocationStrategy: lowest-price
+      ComputeResources:
+        - Name: queue0-compute-resource-0
+          InstanceType: c5n.large
+          MinCount: 0
+          MaxCount: 4
+      Networking:
+        SubnetIds:
+          - subnet-006e97a9837b1710d
```

Comment:
issue(possibly-blocking): hardcoding remote resources makes the tests using these resources fail on a different environment. What do you plan to do against this limitation?

Reply:
Great point, it's either we make it explicit this is bound to our demo environment, or we make the template dynamic

Will work on one of the two options

Reply:
Suggest reaching out to PC team on this one. They have templating working for their integration tests

Reply:
I used [test tags](https://playwright.dev/docs/test-annotations#tag-tests) to explicit the fact that we need a specific environment to run that test

In the future we can adopt something similar to what PC does (as linked by Sean)